pyspark-iris/{{ cookiecutter.repo_name }}/src/requirements.txt

```
@@ -4,10 +4,11 @@ ipython>=7.31.1, <8.0
 isort~=5.0
 jupyter~=1.0
 jupyterlab~=3.0
-kedro[spark.SparkDataSet]=={{ cookiecutter.kedro_version }}
+kedro[spark.SparkDataSet, pandas.CSVDataSet]=={{ cookiecutter.kedro_version }}
```

```suggestion
kedro[spark.SparkDataSet, pandas.ParquetDataSet]=={{ cookiecutter.kedro_version }}
```
Unless `CSVDataSet` is still used somewhere?

---

This makes more sense 👍